packages/client/src/PhantomClient.ts

```diff
@@ -253,12 +307,12 @@ export class PhantomClient {
         walletId: walletId,
         // For EVM transactions, use the object format with kind and bytes
         // For other chains, use the string directly
-        transaction: isEvmTransaction ? { kind: "RLP_ENCODED", bytes: encodedTransaction } : encodedTransaction,
+        transaction: isEvmTransaction ? { kind: "RLP_ENCODED", bytes: preparedTransaction } : preparedTransaction,
```

Comment:
Should we also check that it's a Solana transaction? We'll otherwise be preparing transactions for other chains.